fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -101,5 +101,6 @@
     <Reference Include="UIAutomationTypes" />
     <ProjectReference Include="CSharp_Analysis\CSharp_Analysis.csproj" />
     <ProjectReference Include="..\FSharp.Compiler.Service.ProjectCracker\FSharp.Compiler.Service.ProjectCracker.fsproj" />
+    <ProjectReference Include="$(FSharpSourcesRoot)\fsharp\FSharp.Compiler.Interactive.Settings\FSharp.Compiler.Interactive.Settings.fsproj" />
```

Comment:
@KevinRansom Do you remember why this line was needed? I assume the FCS testing failed if it wasn't present?

(The line causes the FCS build to also build the latest FSharp.Core, which isn't really meant to happen as FCS is only meant to depend on a public lower-versioned FSharp.Core. This is causing some issues in integrations for some of my PRs)

Reply:
@dsyme,  I believe it is in support of the added test to ensure the squiggle is found.
     vsintegration/tests/UnitTests/FsxCompletionProviderTests.fs

Reply:
Hmmm that test isn't included in the FCS testing

Reply:
I suspect it might not be needed, will check